### ISSUE TITLE
Un-selected tabs - 508 contrast issue

### ISSUE THREAD

**Issue by gavinr** [Esri/calcite-web#1106]
Unselected tabs do not have a sufficient contrast ratio

## To Replicate
1. Include CDN version of Calcite on a page
1. Include the tabs sample HTML from http://esri.github.io/calcite-web/documentation/patterns/#tabs (Example: https://jsbin.com/zegesov/edit?html,js,output)
1. Run the Google Chrome Developer Tools Audit (on https://output.jsbin.com/zegesov)
1. See the failing elements under `Accessibility > Contrast`
![image](https://user-images.githubusercontent.com/209355/78145607-9d984900-7420-11ea-9d8c-53c59430dd8f.png)

**Comment by gavinr** [comment]
Moved to https://github.com/Esri/calcite-react/issues/366
